pom.xml

```
@@ -84,11 +84,15 @@
 
   <properties>
     <connid.version>1.4.5.1</connid.version>
-    <google-api-client.version>1.30.10</google-api-client.version>
-    <google-oauth-client.version>1.31.0</google-oauth-client.version>
+    <google-api-client.version>1.33.4</google-api-client.version>
+    <google-oauth-client.version>1.33.2</google-oauth-client.version>
     <google-api-services.version>1.25.0</google-api-services.version>
-    <google-auth-library-oauth2-http.version>0.21.1</google-auth-library-oauth2-http.version>
-    <jackson.version>2.12.6</jackson.version>
+    <google-auth-library-oauth2-http.version>1.6.0</google-auth-library-oauth2-http.version>
+    <jackson.version>2.13.2.2</jackson.version>
+
+    <spring-boot-maven-plugin.version>2.6.5</spring-boot-maven-plugin.version>
```

Latest version is 2.6.6 for both.
```suggestion
    <spring-boot.version>2.6.6</spring-boot.version>
```